web/src/js/components/Dashboard/VideoEngagementComponent.js

```diff
@@ -83,23 +82,31 @@ const VideoEngagementComponent = ({
     <div>
       <div
         ref={poppoverRef}
-        onClick={!adAvailable ? () => setIsPoppoverOpen(true) : undefined}
+        onClick={
+          !adAvailable || !videoAdEligible
```

Comment:
tfac 574

Reply:
@amaliwanag2 I got some bug with state when I completed the 3rd ad and closed it. The button was disabled, but clicking it showed the incorrect dropdown message, then I was unable to close the dropdown:
<img width="531" alt="Screen Shot 2021-10-05 at 3 05 12 PM" src="https://user-images.githubusercontent.com/5924325/136087519-121cd7fb-8555-4b5c-9f58-aa12dd14c97f.png">